lib/blacklight/xml_parser.rb

```diff
@@ -113,7 +113,8 @@ def self.get_master_parent(pre_data, parents_ids, parent_id)
   # non-metadata file.
   ##
   def self.iterate_files(zipfile)
-    files = zipfile.glob("*/**/**")
+    files = zipfile.entries.select(&:file?)
```

Comment:
When I tried with `zipfile.glob("**/**/**")` I was getting directories included. In the documentation it looks like it just maps over all the entries in the zip, not differentiating between files and directories. 
http://www.rubydoc.info/github/rubyzip/rubyzip/Zip/EntrySet#glob-instance_method

So we could do `zipfile.glob('**/**/**').select(&:file?)` instead, but is that different than `zipfile.entries.select(&:file?)`?

I am grabbing all of the files in the root of the export to get the files in the resource directories that are at the root of the export. Currently we are going through our list of files and then deciding which ones to throw away, I.E. *.dat files and things.